inst/rmarkdown/templates/pdf/skeleton/pinp.cls

```diff
@@ -463,10 +518,16 @@ THE ABSTRACT\vskip12pt}\let\go\relax\fi
 	{0pt}
 	{3.25ex plus 1ex minus .2ex}
 	{1.5ex plus .2ex}
+
+
 \newcommand{\showacknow}{% Display acknowledgments section
-\@ifundefined{@acknow}{}{\acknow@section{Acknowledgments}\@acknow}
+\@ifundefined{@acknow}{}{
+\vskip 3.25ex plus 1ex minus .2ex
+\noindent{\sffamily\normalsize\bfseries ACKNOWLEDGMENTS.\hspace{1.5ex plus .2ex}}
```

Comment:
```suggestion
\noindent{\sffamily\normalsize\bfseries Acknowledgements.\hspace{1.5ex plus .2ex}}
```

Reply:
That's good.  I won't commit from the PR.  Let's just collect all "post processing" in one commit.